v2/apiserver/main.go

```diff
@@ -243,6 +245,11 @@ func main() {
 					),
 					Service: eventsService,
 				},
+				&systemREST.SystemEndpoints{
+					APIServerVersion: version.Version(),
+					DatabaseClient:   database.Client(),
+					WriterFactory:    queueWriterFactory,
+				},
```

Comment:
Total nit: The list of endpoints passed to the server was alphabetical prior to this. Might want to move this down, but it's not the end of the world if you don't.